OpenBSD/openbsd-packet-filter/ingest/parser.yml

```
@@ -60,6 +64,10 @@ stages:
           openbsd.pf.routing.flow: "{{parsed_event.message.flow}}"
           openbsd.pf.carp.type: "{{parsed_event.message.type}}"
 
+      - set:
+          error.type: "{{parsed_event.message.transport_type}}"
```

According to the filter, I think that is the value of `error_msg` that is expected here.
```suggestion
          error.type: "{{parsed_event.message.error_msg}}"
```

---

The filter mentions `error_msg` because only when this field is present in the Grok is `transport_type` considered an error type. I already nested `error_msg` in a error.message field (line 37)